src/ethereum/paris/trie.py

```diff
@@ -0,0 +1,469 @@
+"""
+State Trie
+^^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+The state trie is the structure responsible for storing
+`eth1spec.eth_types.Account` objects.
+"""
+
+import copy
+from dataclasses import dataclass, field
+from typing import (
+    Callable,
+    Dict,
+    Generic,
+    List,
+    Mapping,
+    MutableMapping,
+    Optional,
+    Sequence,
+    TypeVar,
+    Union,
+    cast,
+)
+
+from ethereum.crypto.hash import keccak256
+from ethereum.london import trie as previous_trie
```

Comment:
Shouldn't this be `gray_glacier`? I guess the linter didn't get to run yet :rofl: